src/test/java/org/takes/rs/RsPrintTest.java

```diff
@@ -97,6 +100,33 @@ public void closeBodyEvenWhenExceptionHappens() throws IOException {
         );
     }
 
+    @Test
+    public void simple() throws IOException {
+        final StringWriter writer = new StringWriter();
+        final String firstline = "HTTP/1.1 500 Internal Server Error";
+        final Set<String> head = Collections.singleton(firstline);
```

Comment:
@rusefillc Let's use `SetOf` here instead of `Collections.singleton();`; also, you can inline lines 106 and 107 in line 108

Reply:
@paulodamaso please clarify reasoning for SetOf instead of Collections.singleton()?

Reply:
@rusefillc We in `takes` try to follow the ["elegant objects" principles](https://www.elegantobjects.org/), and avoid using static methods and utility classes: they are not proper objects, therefore they don't fit in an object oriented world (read [this](https://www.yegor256.com/2014/05/05/oop-alternative-to-utility-classes.html) for more info).